license-information.md

```diff
@@ -1,29 +1,41 @@
 # License Information
 
-The .NET project uses source and binaries from multiple sources. The MIT license is the primary license used, however, there are some exceptions.
-
-[Windows builds](license-information-windows.md) are the most notable difference, which carry additional license terms.
+The .NET project uses source and binaries from multiple sources that may be important to your use of .NET.
 
 This document is provided for informative purposes only and is not itself a license.
 
 ## Source code
 
-.NET source is held in a variety of [project repos](./Documentation/core-repos.md). They use the MIT license, for example, [dotnet/runtime LICENSE.TXT](https://github.com/dotnet/runtime/blob/main/LICENSE.TXT).
+.NET source uses the MIT license. There are several [project repos](./Documentation/core-repos.md).
+
+Each repo has:
 
-Project repos may include source from other projects, and include a matching third-party notice for copied and modified source, for example [dotnet/runtime THIRD-PARTY-NOTICES.TXT](https://github.com/dotnet/runtime/blob/main/THIRD-PARTY-NOTICES.TXT).
+- A license, for example, [dotnet/runtime LICENSE.TXT](https://github.com/dotnet/runtime/blob/main/LICENSE.TXT).
+- Third party notice file, for example, [dotnet/runtime THIRD-PARTY-NOTICES.TXT](https://github.com/dotnet/runtime/blob/main/THIRD-PARTY-NOTICES.TXT)
+
+Third party notice files are used to document source copied (and possibly further modified) from other projects.
 
 [Project copyright guidance](https://github.com/dotnet/runtime/blob/main/docs/project/copyright.md) provides more details on our policies.
 
 ## Binaries
 
-Project binaries (like the .NET runtime distribution) primarily use the MIT license. Binaries built for macOS and Linux exclusively use the MIT license.
+Microsoft built binaries use the MIT license, for:
+
+- NuGet packages

```

Comment:
Runtime is delivered via NuGet packages too, so this description is ambiguous.

https://github.com/dotnet/runtime/blob/main/docs/project/licensing-assets.md solves this problem by talking about product distributions and package distributions. Should we do the same here?

Reply:
Yes. I paused at that when I wrote it. I'll fix it.